praw/models/reddit/subreddit.py

```diff
@@ -404,6 +405,31 @@ def quaran(self):
         """
         return SubredditQuarantine(self)
 
+    @cachedproperty
+    def rules(self):
+        """Provide an instance of :class:`.SubredditRules`.
+
+        Use this attribute for interacting with a subreddit's rules.
+        For example, to list all the rules for a subreddit:
+
+        .. code-block:: python
+
+            for rule in reddit.subreddit('AskReddit').rules:
+                print(rule)
+
+        You can also add rules to the subreddit, if you are a moderator. For
+        example, to make a rule called ``"No spam"`` in the subreddit ``"NAME"``:
+
+        .. code-block:: python
+
+           reddit.subreddit('NAME').rules.mod.add(
+               "No spam",
```

Comment:
It's not clear from here what these positional arguments are. Can we use keyword arguments in the example please?